pkg/trimaran/resourcestats.go

```diff
@@ -121,7 +121,7 @@ func GetResourceLimits(pod *v1.Pod) *framework.Resource {
 	})
 }
 
-// GetEffectiveResource: calculate effective resources of a pod (CPU and Memory)
+// GetEffectiveResource : calculate effective resources of a pod (CPU and Memory)
```

Comment:
why the extra space?

Reply:
I noticed that the method annotations in this directory were all like that. I simply reverted the changes to maintain code cleanliness. I have already rolled back the changes.
https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/pkg/trimaran/resourcestats.go#L77
```go
// GetResourceRequested : calculate the resource requests of a pod (CPU and Memory)
func GetResourceRequested(pod *v1.Pod) *framework.Resource {
	return GetEffectiveResource(pod, func(container *v1.Container) v1.ResourceList {
		return container.Resources.Requests
	})
}

// GetResourceLimits : calculate the resource limits of a pod (CPU and Memory)
func GetResourceLimits(pod *v1.Pod) *framework.Resource {
	return GetEffectiveResource(pod, func(container *v1.Container) v1.ResourceList {
		return container.Resources.Limits
	})
}

```